docs/architecture/cloud-native/defining-cloud-native.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Defining Cloud Native
```

Comment:
why this content is being moved to the original file names? the reason why they don't match exactly the title is because of SEO. Here the new filename breaks two rules:
- uses ing
- repeats words several times in the path